sass/modules/tooltip.sass

```diff
@@ -1,3 +1,22 @@
+// Compatibility tweaks for v4
+  .tooltip-arrow
+    position: absolute
+    width: 0
+    height: 0
+    border-color: transparent
+    border-style: solid
+
+.tooltip.bottom
+  margin-top: 3px
+  padding: 5px 0
+
+.tooltip.bottom .tooltip-arrow
```

Comment:
Can we nest this with the selector above?

Reply:
done!